service/src/main/java/uk/gov/hmcts/reform/fpl/config/LocalAuthorityNameLookupConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import com.google.common.collect.ImmutableMap;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Arrays;
+import java.util.Map;
+
+@Configuration
+public class LocalAuthorityNameLookupConfiguration {
+
+    private static final String MAPPINGS_SEPARATOR = ";";
+    private static final String ENTRY_SEPARATOR = "=>";
+
+    private Map<String, String> mapping;
```

Comment:
Let's make it final. I like to have class level fields final making sure no-one will ever override reference.